cmd/configschema/go.mod

```diff
@@ -16,6 +16,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/logicmonitor/lm-data-sdk-go v0.7.1 // indirect
+
```

Comment:
Could you join this with the main `require` block for `// indirect` dependencies just below?

Reply:
sure will do that!